src/editor.html

```diff
@@ -952,9 +957,9 @@
 
 				var offsetY = window.document.body.scrollTop;
 
-				var footer = $('#zss_editor_footer');
+				var footer = document.getElementById('zss_editor_footer');
```

Comment:
I did it because I was needed offsetTop and offsetHeight in my calculations and It's easier to take it directly from node. And also I noticed that inside editor there are other places where we use document methods directly, so I decided that I will not break any code style in this context.